### ISSUE TITLE
It looks like rgba(hex, 0) doesn't get processed

### ISSUE THREAD

**Issue by eliekhoury** [avanes/postcss-color-alpha#10]
Is rgba(hex, 0) not supported for a reason or is it a bug?

https://github.com/avanes/postcss-color-alpha/blob/master/index.js#L144

Thanks.

**Comment by avanes** [comment]
Hi, @eliekhoury. I've added test for this case and it's passing
https://github.com/avanes/postcss-color-alpha/blob/master/test/test.js#L70-L72

**Comment by avanes** [comment]
May be it's not working as expected in other context. What is your case?

**Comment by eliekhoury** [comment]
Here's my case. `rgba(#fff, 0)` in the context below is not transformed.

```
background-image: linear-gradient(to bottom, rgba(#fff, .6) 0%, rgba(#fff, 0), 100%);
```

**Comment by avanes** [comment]
@eliekhoury thank you! Published 0.1.1 ;)

**Comment by eliekhoury** [comment]
Thanks!
